app.js

```diff
@@ -93,6 +59,48 @@ router.post('/account', async ctx => {
     ctx.body = response;
 });
 
+const password = require('secure-random-password');
+const models = require('./models');
+const FROM_PHONE = '+14086179592';
+router.post('/account/:phoneNumber/:accountId/requestCode', async ctx => {
+    const accountId = ctx.params.accountId;
+    const phoneNumber = ctx.params.phoneNumber;
+
+    const securityCode = password.randomPassword({ length: 6, characters: password.digits });
```

Comment:
define 6 as a const? (also is 6 too few chars?)

Reply:
Done